contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -185,181 +197,203 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
-        authP(MANAGE_SIGNING_KEYS, arr(_operator_id))
-    {
+     * @notice Add `_quantity` validator signing keys of operator #`_id` to the set of usable keys. Concatenated keys are: `_pubkeys`. Can be done by the DAO in question by using the designated rewards address.
+     * @dev Along with each key the DAO has to provide a signatures for the
+     *      (pubkey, withdrawal_credentials, 32000000000) message.
+     *      Given that information, the contract'll be able to call
+     *      deposit_contract.deposit on-chain.
+     * @param _operator_id Node Operator id
+     * @param _quantity Number of signing keys provided
+     * @param _pubkeys Several concatenated validator signing keys
+     * @param _signatures Several concatenated signatures for (pubkey, withdrawal_credentials, 32000000000) messages
+     */
+    function addSigningKeys(
+        uint256 _operator_id,
+        uint256 _quantity,
+        bytes _pubkeys,
+        bytes _signatures
+    ) external authP(MANAGE_SIGNING_KEYS, arr(_operator_id)) {
         _addSigningKeys(_operator_id, _quantity, _pubkeys, _signatures);
     }
 
     /**
-      * @notice Add `_quantity` validator signing keys of operator #`_id` to the set of usable keys. Concatenated keys are: `_pubkeys`. Can be done by node operator in question by using the designated rewards address.
-      * @dev Along with each key the DAO has to provide a signatures for the
-      *      (pubkey, withdrawal_credentials, 32000000000) message.
-      *      Given that information, the contract'll be able to call
-      *      deposit_contract.deposit on-chain.
-      * @param _operator_id Node Operator id
-      * @param _quantity Number of signing keys provided
-      * @param _pubkeys Several concatenated validator signing keys
-      * @param _signatures Several concatenated signatures for (pubkey, withdrawal_credentials, 32000000000) messages
-      */
+     * @notice Add `_quantity` validator signing keys of operator #`_id` to the set of usable keys. Concatenated keys are: `_pubkeys`. Can be done by node operator in question by using the designated rewards address.
+     * @dev Along with each key the DAO has to provide a signatures for the
+     *      (pubkey, withdrawal_credentials, 32000000000) message.
+     *      Given that information, the contract'll be able to call
+     *      deposit_contract.deposit on-chain.
+     * @param _operator_id Node Operator id
+     * @param _quantity Number of signing keys provided
+     * @param _pubkeys Several concatenated validator signing keys
+     * @param _signatures Several concatenated signatures for (pubkey, withdrawal_credentials, 32000000000) messages
+     */
     function addSigningKeysOperatorBH(
         uint256 _operator_id,
         uint256 _quantity,
         bytes _pubkeys,
         bytes _signatures
-    )
-        external
-    {
+    ) external {
         require(msg.sender == operators[_operator_id].rewardAddress, "APP_AUTH_FAILED");
```

Comment:
Interesting: in the case of `rewardAddress` collision, one node operator can submit keys for another, though we don't enforce its uniqueness either on-chain or by the policies (If I am not wrong)

Reply:
decided it's ok